scenarios/18_Start_of_the_Quest.cfg

```diff
@@ -420,12 +432,16 @@
             name=_"Hans"
             x=12
             y=22
-            experience={ON_DIFFICULTY 8 7 6} # out of 23
+            experience={ON_DIFFICULTY4 8 7 6 5} # out of 23
             side=1
             [modifications]
                 {TRAIT_LOYAL}
                 {TRAIT_STRONG}
+#ifdef NIGHTMARE
```

Comment:
And so on.

Reply:
[086c1d2](https://github.com/nemaara/A_New_Order/pull/53/commits/086c1d25c13ea2fc5549ae746b68eb2476cc283a).